docs/streams/upgrade-guide.html

```diff
@@ -135,6 +135,12 @@ <h3 class="anchor-heading"><a id="streams_notable_changes" class="anchor-link"><
 
     <h3><a id="streams_api_changes_400" href="#streams_api_changes_400">Streams API changes in 4.0.0</a></h3>
 
+    <p>
+        In this release two configs default.deserialization.exception.handler and default.production.exception.handler are deprecated, as they don't have any overwrites, which is described in
+        <a href="https://cwiki.apache.org/confluence/display/KAFKA/KIP-1056%3A+Remove+%60default.%60+prefix+for+exception+handler+StreamsConfig">KIP-1056</a>
+        You can refer to new configs via <code>StreamsConfig#DESERIALIZATION_EXCEPTION_HANDLER_CLASS_CONFIG</code> and <code>StreamsConfig#PRODUCTION_EXCEPTION_HANDLER_CLASS_CONFIG</code>.
```

Comment:
I think it's better to use config name `<code>deserialization.exception.handler</code>` directly, and not refer to `StreamsConfig` -- in the end, the config name changed, what is the central thing (eg, for config files) -- it's only a "side effect" that we also add new variables to `StreamsConfig`.